generatejson.py

```diff
@@ -49,15 +49,18 @@ def getpkginfopath(filename):
                             stdout=subprocess.PIPE,
                             stderr=subprocess.PIPE)
     (bom, err) = proc.communicate()
-    bom = bom.strip().split('\n')
-    if proc.returncode == 0:
+    
+    try:
+        # Decode the byte stream and convert to list
+        bom = bom.decode("utf-8").split()
```

Comment:
Decode the byte stream and convert to list that we can then parse and look for the things that we need.